### PR TITLE
PORT-6432 Create and delete cluster action guide

### PR DESCRIPTION
**Pull request by phalbert**
# Description

A guide to create and manage k8s clusters from Port.

## Added docs pages

Please also include the path for the added docs

- Creating and managing Kubernetes clusters (`/create-self-service-experiences/setup-backend/github-workflow/examples/manage-clusters.md`)

### CODE REVIEW

File: docs/create-self-service-experiences/setup-backend/github-workflow/examples/manage-clusters.md [hunk 1]
```diff
@@ -0,0 +1,528 @@
+---
+sidebar_position: 3
+---
+
+import PortTooltip from "/src/components/tooltip/tooltip.jsx";
+
+# Creating and managing Kubernetes clusters
+
+This guide provides a step-by-step walkthrough on creating Kubernetes clusters with the use of Crossplane and ArgoCD.
```
[the remark this answers is not in the document]
Comment (phalbert) [reply docs/create-self-service-experiences/setup-backend/github-workflow/examples/manage-clusters.md:9]: resolved

File: docs/create-self-service-experiences/setup-backend/github-workflow/examples/manage-clusters.md [hunk 2]
```diff
@@ -0,0 +1,528 @@
+---
+sidebar_position: 3
+---
+
+import PortTooltip from "/src/components/tooltip/tooltip.jsx";
+
+# Creating and managing Kubernetes clusters
```
[the remark this answers is not in the document]
Comment (phalbert) [reply docs/create-self-service-experiences/setup-backend/github-workflow/examples/manage-clusters.md:7]: resolved

File: docs/create-self-service-experiences/setup-backend/github-workflow/examples/manage-clusters.md [hunk 3]
```diff
@@ -0,0 +1,528 @@
...
+
+## Prerequisites
+
+1. A Control plane that will be used to create clusters and other infrastructure. We will use [crossplane](https://docs.crossplane.io/latest/software/install/).
+2. A GitOps operator for automatically running operations on our cluster based on changes in our manifests repository. We will be using [ArgoCD](https://argo-cd.readthedocs.io/en/stable/).
+3. [Install Helm](https://helm.sh/docs/intro/install/)
+4. A GitHub repository to contain your resources i.e. the github workflow file, port resources, and infrastructure manifests.
+5. Prior knowledge of Port Actions is essential for following this guide. Learn more about them [here](/create-self-service-experiences/setup-ui-for-action/).
+
+:::tip
```
[the remark this answers is not in the document]
Comment (phalbert) [reply docs/create-self-service-experiences/setup-backend/github-workflow/examples/manage-clusters.md:19]: resolved

File: docs/create-self-service-experiences/setup-backend/github-workflow/examples/manage-clusters.md [hunk 4]
```diff
@@ -0,0 +1,528 @@
...
+Copy the `crossplane` folder from our examples repository [here](https://github.com/port-labs/self-service-actions-examples) to follow along through the guide.
+:::
+
+## 1. Control Plane Setup
+
+:::tip TIP
```
[the remark this answers is not in the document]
Comment (phalbert) [reply docs/create-self-service-experiences/setup-backend/github-workflow/examples/manage-clusters.md:25]: alright

File: docs/create-self-service-experiences/setup-backend/github-workflow/examples/manage-clusters.md [hunk 5]
```diff
@@ -0,0 +1,528 @@
...
+      selfHeal: true
+      prune: true
+      allowEmpty: true
+```
+</details>
+
+
+## 5. Let's test it.
+
+- On the [self-service](https://app.getport.io/self-serve) screen, go to the create cluster action and fill in the cluster properties.
```
[the remark this answers is not in the document]
Comment (phalbert) [reply docs/create-self-service-experiences/setup-backend/github-workflow/examples/manage-clusters.md]: resolved

**Review by hadar-co — APPROVED** [review]

**Pull request merged**